certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2953 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PPM Express",
+    "version": "v1.0",
+    "description": "PPM Express is Online Project Portfolio Management Software that was designed to easily integrate and orchestrate your existing projects in Microsoft Planner, Azure DevOps, Jira Software and Microsoft Project Online.",
+    "contact": {
+      "name": "PPM Express Support",
+      "url": "https://ppm.express/contact-us/",
+      "email": "support@ppm.express"
+    }
+  },
+  "host": "api-us.ppm.express",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-capabilities": {
+    "testConnection": {
+      "operationId": "Me_Get"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://ppm.express"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://ppm.express/privacy-policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Business Management;Business Intelligence"
+    }
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/Challenges/{id}": {
+      "get": {
+        "tags": [
+          "Challenges"
+        ],
+        "summary": "Get a Challenge",
+        "description": "Gets an existing Challenge by Id",
+        "operationId": "Challenges_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/challenge_id__in_path"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns existing item",

```

Comment:
@vsorokin-ppmexpress please make a bit more precise descriptions for response codes in your operations. E.g. existing item returned, item not found, user not authorized, operation not allowed, etc. The 'if' word is not necessary here.